drivers/sensor/lsm6dsl/lsm6dsl_spi.c

```diff
@@ -30,12 +30,17 @@ static struct spi_config lsm6dsl_spi_conf = {
 static int lsm6dsl_raw_read(struct spi_config *spi_cfg, u8_t reg_addr,
```

Comment:
Should be:
static int lsm6dsl_raw_read(struct lsm6dsl_data *data, u8_t reg_addr,